<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyBase</class>
 <widget class="QMainWindow" name="PyBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <item row="1" column="0">
     <widget class="QFrame" name="frBase">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="2" column="0" rowspan="2">
        <widget class="QFrame" name="frMenuLateral">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btHide">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Menu</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt2">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt3">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btSettings">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QFrame" name="frContent">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QFrame" name="frame">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="pushButton">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>20</y>
               <width>81</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
            <widget class="QScrollBar" name="horizontalScrollBar">
             <property name="geometry">
              <rect>
               <x>130</x>
               <y>20</y>
               <width>160</width>
               <height>16</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
            <widget class="QSlider" name="horizontalSlider">
             <property name="geometry">
              <rect>
               <x>310</x>
               <y>20</y>
               <width>160</width>
               <height>22</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
            <widget class="QSlider" name="verticalSlider">
             <property name="geometry">
              <rect>
               <x>120</x>
               <y>50</y>
               <width>22</width>
               <height>160</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
            <widget class="QScrollBar" name="verticalScrollBar">
             <property name="geometry">
              <rect>
               <x>70</x>
               <y>50</y>
               <width>16</width>
               <height>160</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
            <widget class="QTableWidget" name="tableWidget">
             <property name="geometry">
              <rect>
               <x>230</x>
               <y>60</y>
               <width>421</width>
               <height>131</height>
              </rect>
             </property>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <row>
              <property name="text">
               <string>New Row</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>New Row</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>New Row</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>New Row</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>New Row</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>New Row</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>New Row</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>New Row</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <item row="1" column="0">
              <property name="text">
               <string>asdas</string>
              </property>
             </item>
             <item row="1" column="1">
              <property name="text">
               <string>dasd</string>
              </property>
             </item>
             <item row="1" column="2">
              <property name="text">
               <string>asda</string>
              </property>
             </item>
             <item row="1" column="3">
              <property name="text">
               <string>sdasda</string>
              </property>
             </item>
             <item row="2" column="0">
              <property name="text">
               <string>sda</string>
              </property>
             </item>
             <item row="2" column="1">
              <property name="text">
               <string>sdasd</string>
              </property>
             </item>
             <item row="2" column="2">
              <property name="text">
               <string>as</string>
              </property>
             </item>
             <item row="2" column="3">
              <property name="text">
               <string>dasdas</string>
              </property>
             </item>
             <item row="3" column="0">
              <property name="text">
               <string>dasd</string>
              </property>
             </item>
             <item row="3" column="1">
              <property name="text">
               <string>asd</string>
              </property>
             </item>
             <item row="3" column="2">
              <property name="text">
               <string>asd</string>
              </property>
             </item>
             <item row="3" column="3">
              <property name="text">
               <string>asdasd</string>
              </property>
             </item>
             <item row="4" column="0">
              <property name="text">
               <string>asdas</string>
              </property>
             </item>
             <item row="4" column="1">
              <property name="text">
               <string>dasd</string>
              </property>
             </item>
             <item row="4" column="2">
              <property name="text">
               <string>asd</string>
              </property>
             </item>
             <item row="4" column="3">
              <property name="text">
               <string>asda</string>
              </property>
             </item>
             <item row="5" column="0">
              <property name="text">
               <string>asd</string>
              </property>
             </item>
             <item row="5" column="1">
              <property name="text">
               <string>asd</string>
              </property>
             </item>
             <item row="5" column="2">
              <property name="text">
               <string>asd</string>
              </property>
             </item>
             <item row="5" column="3">
              <property name="text">
               <string>sad</string>
              </property>
             </item>
             <item row="6" column="0">
              <property name="text">
               <string>asd</string>
              </property>
             </item>
             <item row="6" column="1">
              <property name="text">
               <string>asd</string>
              </property>
             </item>
             <item row="6" column="2">
              <property name="text">
               <string>asd</string>
              </property>
             </item>
             <item row="7" column="0">
              <property name="text">
               <string>asd</string>
              </property>
             </item>
             <item row="7" column="1">
              <property name="text">
               <string>asd</string>
              </property>
             </item>
             <item row="7" column="2">
              <property name="text">
               <string>asd</string>
              </property>
             </item>
             <item row="7" column="3">
              <property name="text">
               <string>asd</string>
              </property>
             </item>
            </widget>
            <widget class="QGroupBox" name="groupBox">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>230</y>
               <width>120</width>
               <height>80</height>
              </rect>
             </property>
             <property name="title">
              <string>GroupBox</string>
             </property>
            </widget>
            <widget class="QFrame" name="frame_2">
             <property name="geometry">
              <rect>
               <x>220</x>
               <y>230</y>
               <width>120</width>
               <height>80</height>
              </rect>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit">
             <property name="geometry">
              <rect>
               <x>420</x>
               <y>220</y>
               <width>231</width>
               <height>21</height>
              </rect>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QTextEdit" name="textEdit">
             <property name="geometry">
              <rect>
               <x>420</x>
               <y>260</y>
               <width>231</width>
               <height>171</height>
              </rect>
             </property>
            </widget>
            <widget class="QSpinBox" name="spinBox">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>390</y>
               <width>42</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QDoubleSpinBox" name="doubleSpinBox">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>390</y>
               <width>62</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QRadioButton" name="radioButton">
             <property name="geometry">
              <rect>
               <x>220</x>
               <y>330</y>
               <width>82</width>
               <height>18</height>
              </rect>
             </property>
             <property name="text">
              <string>RadioButton</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="checkBox">
             <property name="geometry">
              <rect>
               <x>220</x>
               <y>370</y>
               <width>70</width>
               <height>18</height>
              </rect>
             </property>
             <property name="text">
              <string>CheckBox</string>
             </property>
            </widget>
            <widget class="QComboBox" name="comboBox">
             <property name="geometry">
              <rect>
               <x>220</x>
               <y>400</y>
               <width>151</width>
               <height>22</height>
              </rect>
             </property>
             <item>
              <property name="text">
               <string>New Item</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>New Item</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>New Item</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>New Item</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>New Item</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>New Item</string>
              </property>
             </item>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QFrame" name="frTitleBar">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frIcon">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelAppName">
            <property name="text">
             <string>App</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>371</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btMin">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Minimizar</string>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/16x16/16x16/cil-minus.png</normaloff>:/16x16/16x16/cil-minus.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btMax">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Maximizar</string>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/16x16/16x16/cil-media-stop.png</normaloff>:/16x16/16x16/cil-media-stop.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btClose">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <stylestrategy>PreferAntialias</stylestrategy>
             </font>
            </property>
            <property name="toolTip">
             <string>Fechar</string>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/16x16/16x16/cil-x.png</normaloff>:/16x16/16x16/cil-x.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QFrame" name="frStatusBar">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelDevelop">
            <property name="text">
             <string>develop</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>487</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelVersion">
            <property name="text">
             <string>version</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frGrip">
            <property name="minimumSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">/*QFrame{
	background-image: url(:/16x16/16x16/cil-size-grip.png);
    background-repeat: no-repeat;
	background-color: transparent;
    background-position: right bottom;
    margin: 3px 1px;
}*/</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
